bot.ts

```diff
@@ -36,90 +57,158 @@ const commands: SlashCommandPartial[] = [
+			);
+			return;
+		}
+
+		await i.reply(
+			game.discord
+				? `Here's the invite to \`${game.names.international}\`: ${game.discord}`
+				: `Couldn't find any Discord invite for \`${game.names.international}\`. Here's a link to their forums: ${game.weblink}/forums`,
+		);
 	}
 
 	@slash("series-community")
-	async seriesCommunity(t: ApplicationCommandInteraction) {
-		await t.defer();
+	async seriesCommunity(i: ApplicationCommandInteraction) {
+		await i.defer();
 		const res = await fetch(
-			`https://www.speedrun.com/api/v1/series/${t.option("series")}`,
+			`https://www.speedrun.com/api/v1/series/${i.option("series")}`,
 		);
-		const series = (await res.json()).data as SearchResult;
-
-		if (series) {
-			await t.reply({
-				content: series.discord
-					? `Here's the invite to the \`${series.names.international} series\`: ${series.discord}`
-					: `Couldn't find any invite for the \`${series.names.international} series\`, here's a link to their forums: ${series.weblink}/forum`,
-			});
-		} else {
-			await t.reply({
-				content: `\`${t.option("series")} series\` not found`,
-			});
+		const series: v1SearchResult | undefined = (await res.json()).data;
+		const userMention = i.option("mention");
+	
+		if (!series) {
+			await i.reply(
+				userMention
+					? `${userMention}, the \`${i.option("series")} Series\` wasn't found`
+					: `\`${i.option("series")} Series\` not found`
+			);
+			return;
+		}
+
+		// series.weblink doesn't have the "/series/" prefix
+		const formLink = `https://www.speedrun.com/series/${series.abbreviation}/forums`;
+
+		if (userMention) {
+			await i.reply(
+				series.discord
+					? `${userMention}, here's the invite to the \`${series.names.international} Series\`: ${series.discord}`
+					: `${userMention}, couldn't find any Discord invite for the \`${series.names.international} Series\`. Here's a link to their forums: ${formLink}`,
+			);
+			return;
 		}
+
+		await i.reply(
+			series.discord
+				? `Here's the invite to the \`${series.names.international} Series\`: ${series.discord}`
+				: `Couldn't find any Discord invite for the \`${series.names.international} Series\`. Here's a link to their forums: ${formLink}`,
+		);
+	}
+
+	@slash()
+	async github(i: ApplicationCommandInteraction) {
+		await i.reply("Here's the link to the GitHub repository: https://github.com/AnInternetTroll/speedruncom-discord-bot");
 	}
 
 	@autocomplete("*", "*")
-	async Autocomplete(d: AutocompleteInteraction) {
-		const completions: ApplicationCommandChoice[] = [];
-		const isGame = Boolean(d.option("game"));
-		const res = await fetch(
+	async myAutocomplete(d: AutocompleteInteraction) {
+		if (!d.focusedOption.value) {
+			await d.autocomplete([]);
+			return;
+		}
+
+		const isGame = Boolean(d.focusedOption.name == "game");
+
+		const v2Res = await fetch(`https://www.speedrun.com/api/v2/GetSearch?_r=${
```

Comment:
I think I've had some issues in the past with non-base64-able characters here. I haven't tested this yet, but I think POST would be better than GET.

Unless GET calls are cached better on speedrun.com's side, but I am not too sure of that

Reply:
I was also gonna mention the v2 library, but it also does GET calls here so guilty :P

https://git.sr.ht/~aninternettroll/speedruncomapiv2/tree/master/item/src/GetSearch/GetSearch.ts

https://git.sr.ht/~aninternettroll/speedruncomapiv2/tree/master/item/src/utils/requests.ts#L123

Reply:
I didn't realise POST worked on that endpoint, since the site uses GET itself. But I've now changed it